src/muz/spacer/spacer_util.cpp

```diff
@@ -69,6 +69,25 @@ Revision History:
 
 namespace spacer {
 
+    namespace contains_def_ns {
```

Comment:
there must be a more generic way to do this, using some support in z3. I don't like adding exceptions to the code